### PR TITLE
Autoscale in batches

### PR DESCRIPTION
**Pull request by obino**
This PR allows for the system to react more aggressively when there are backlogs on the applications. It will start more than one AppServer each duty cycle.

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2619/ this was before merging master.

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2620/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2629/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2634/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -5515,39 +5499,60 @@ def try_to_scale_up(app_name)
           Djinn.log_debug("#{host} CPUs are too busy.")
           break
         end
-        available_hosts << host
+
+        # We add the host as many times as AppServers it can run.
+        (max_new_tot > max_new_free ? max_new_free : max_new_tot).downto(1) {
```
Comment (tmarballi) [inline AppController/djinn.rb]: Would you mind renaming the max_new_tot variable to max_new_total, I usually prefer using the full word if its not too long to begin with.

**Pull request merged**